### PR TITLE
clingen-anyvar helm chart

### LINKED ISSUE #28: Create a helm chart for anyvar

### PR DESCRIPTION
**Pull request by sjahl**
This changeset creates a parameterized helm chart for the anyvar service. It doesn't look like this service is deployed in more than one environment, but it was a simple enough manifest that I thought it was a good place to start.

I've chosen some parameters that seem the most valuable to parameterize, I'll be looking for feedback from the dev team to see what else they think might be worth parameterizing.

Closes #28

### CODE REVIEW

File: helm/charts/clingen-anyvar/values.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+seqrepo_version: &seqrepo_vers 2020-04-13
+seqrepo_docker_tag: *seqrepo_vers
```
Comment (theferrit32) [inline helm/charts/clingen-anyvar/values.yaml:2]: Interesting, I have never seen this YAML feature before

Comment (sjahl) [reply helm/charts/clingen-anyvar/values.yaml:2]: Those are YAML anchors: https://support.atlassian.com/bitbucket-cloud/docs/yaml-anchors/

I've found them useful in a few cases so far where I need to re-use a string value. In this case, I'm defaulting to keeping the docker image tag in sync with the seqrepo version, but still allows the image tag to be overridden if needed.

**Comment by sjahl** [comment]
I think this one's actually ready for a review. Since there's only a deployment for this one, and it's only in dev, there's not much to it. Please let me know if you have questions, or if there's anything I can clarify.

You can compare what this generates with the current manifest by cd'ing into the `helm/` folder and running a `helm template clingen-anyvar charts/clingen-anyvar`

File: helm/charts/clingen-anyvar/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,73 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: anyvar
+  labels:
+    app: anyvar
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/part-of: {{ .Chart.Name }}
+    helm.sh/chart: {{ .Chart.Name }}
```
Comment (theferrit32) [inline helm/charts/clingen-anyvar/templates/deployment.yaml:10]: Are these just required to be added in order for helm to recognize the deployment once it's in the kubernetes cluster?

Comment (sjahl) [reply helm/charts/clingen-anyvar/templates/deployment.yaml:10]: I don't think they're required, but it's a best practice to help determine which components are part of which chart. I might actually be able to create a helper that sets these on everything, which should help reduce some of the repetitiveness.

**Review by theferrit32 — APPROVED** [review]

**Pull request merged**